apps/web/src/App.tsx

```diff
@@ -49,6 +48,8 @@ import { SnippetPage } from './pages/templates/components/SnippetPage';
 import { TemplateEditor } from './pages/templates/components/TemplateEditor';
 import { ProvidersPage } from './pages/templates/components/ProvidersPage';
 import { InAppSuccess } from './pages/quick-start/steps/InAppSuccess';
+// import { IntegrationsStore } from './pages/integrations/IntegrationsStorePage';
```

Comment:
commented the old code as we will need it when we will implement the feature flag